lib/src/signed_video_internal.h

```diff
@@ -144,7 +144,7 @@ struct _signed_video_t {
   bool is_golden_sei;  // Flag that tells if a SEI is a golden SEI
   bool using_golden_sei;  // Flag that tells if golden SEI prinsiple is used
   bool signing_started;
-  bool is_first_triggered_signing;
+  bool generate_sei;
```

Comment:
I think this flag now deserves an explanation since it can be interpreted as controlling SEIs in general.

Reply:
Maybe the explanation got a little bit too detailed (sorry). It would have been enough to say "Flips to true after the first signing attempt".

I would probably have named it `first_signing_attempt` and set it to `true` at create time. Then flip it to `false` after the first attempt.